roles/telemetry_verify_metrics/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Verify OpenStack is deployed correctly
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_cr_test_id: RHOSO-1258
+    common_cr_ready_test_id: RHOSO-1259
+    common_cr_list:
+      - kind: telemetry
+        name: telemetry
+        condition_type: Ready
```

Comment:
I like the use of dicts here, which makes the test configs more readable than using lists.